assets/airports/egnm.json

```diff
@@ -10,251 +10,2015 @@
-        "GAM18": ["N53d30m18.00",  "W001d17m07.00"],
-        "GAM15": ["N53d28m74.00",  "W001d12m16.00"],
-        "DOPEK": ["N53d23m04.00",  "W000d55m32.00"],
-        "LAMIX": ["N53d18m48.00",  "W000d40m93.00"],
-        "D140B": ["N53d50m14.00",  "W001d36m85.00"],
-        "GAM36": ["N53d46m54.00",  "W001d30m90.00"],
-        "GAM32": ["N53d43m26.00",  "W001d27m08.00"],
-        "GAM29": ["N53d40m79.00",  "W001d24m22.00"],
-        "GA19A": ["N53d32m56.00",  "W001d14m72.00"],
-        "GA15A": ["N53d29m26.00",  "W001d10m93.00"],
-        "GAM14": ["N53d28m44.00",  "W001d09m99.00"],
-        "LBA"  : ["N53d51m54.00",  "W001d39m10.00"]
+        "_GAM15DME": ["N53d28m25", "W001d12m42"],
+        "_GAM319018": [53.50334238484, -1.28370756576],
+        "_GAM319019": [53.51564501839, -1.30250407183],
+        "_GAM319031": [53.66304347462, -1.52891347974],
+        "_GAM319035": [53.71208001909, -1.60473424361],
+        "_GAM319038": [53.74882567603, -1.66171547265],
+        "_GAM327014": [53.47423039856, -1.16582663874],
+        "_GAM327015": [53.48798922439, -1.18151693089],
+        "_GAM327019": [53.54300356034, -1.24437981498],
+        "_GAM327029": [53.68039210836, -1.40225184649],
+        "_GAM327032": [53.72156739109, -1.44981368631],
+        "_GAM327036": [53.77643789435, -1.51337415705],
+        "_GAM327POL118": [53.45102, -1.13938],
+        "_GAMR319INTPOLR118": [53.48876, -1.26144],
+        "_ILBF2DME": [53.83349605283, -1.61072235541],
+        "_ILF0.5DME": [53.88059032404, -1.68318520718],
+        "_ILF2.1DME": ["N53d53m40", "W001d42m58"],
+        "_ILF3.5DME": ["N53d54m05", "W001d45m21"],
+        "_INTGAM319": [53.83812565585, -1.80078584616],
+        "_INTGAM327": [53.80386036283, -1.54521643089],
+        "_INTLBA272": [53.86807087448, -1.99056411465],
+        "_POL053010": [53.84593149446, -1.88134211524],
+        "_POL053011": [53.85611304431, -1.85908369973],
+        "_POL053012": [53.86629039438, -1.83681441],
+        "_POL083010": [53.76661345001, -1.82518687127],
+        "_POL083014": [53.77552254703, -1.7138449063],
+        "_POL118035": [53.47461817874, -1.22387498779],
+        "BARTN": ["N53d28m14", "W002d25m41"],
+        "BATLI": ["N53d38m45", "W001d21m37"],
+        "CALDA": ["N53d46m27", "W002d38m38"],
+        "DENBY": ["N53d31m13", "W001d57m40"],
+        "DOPEK": ["N53d23m04", "W000d55m32"],
+        "EMBOR": ["N52d57m48", "W001d17m45"],
+        "GAM"  : ["N53d16m53", "W000d56m50"],
+        "GASKO": ["N54d13m29", "W001d57m21"],
+        "GOLES": ["N53d36m29", "W001d05m00"],
+        "LAMIX": ["N53d18m48", "W000d40m93"],
+        "LBA"  : ["N53d51m54", "W001d39m10"],
+        "NELSA": ["N53d51m48", "W002d11m05"],
+        "POL"  : ["N53d44m38", "W002d06m12"],
+        "REXAM": ["N53d04m00", "W003d09m37"],
+        "TNT"  : ["N53d03m14", "W001d40m12"],
+        "WAL"  : ["N53d23m31", "W003d08m04"]
     },
-
+    "restricted": [
+        {
+            "name": "R321",
```

Comment:
I'm not sure if we should even include R321 since it applies to helicopters only. Although it doesn't really affect the flow of traffic so :man_shrugging: